base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -1061,6 +1061,23 @@
 					</macro>
 				</macros>
 			</value>
+		</entry>
+		<entry>
+			<key>Fermi chopper (Digital Drive)</key>
+			<value>
+				<categories>
+					<category>Choppers</category>
+				</categories>
+				<type>CHOPPER</type>
+				<path>FZJ_DD_Fermi_Chopper.opi</path>
+				<description>The OPI for the FZJ Digital Drive Fermi Chopper.</description>
+				<macros>
+					<macro>
+						<name>FZJDDFCH</name>
+						<description>The fermi chopper PV prefix (e.g. FZJDDFCH_01).</description>
+					</macro>
```

Comment:
Why do you need this macro? There's only 1 `FZJDDCH` IOC

Reply:
Used existing Fermi Chopper OPI as template so assumed it met standards.  However, all cosmetic issues addressed.
Followed convention of macros for IOC names, but superfluous macro now removed.